### ISSUE TITLE
Docker on ubuntu--groups: cannot find name for group

### ISSUE THREAD

**Issue by hanm2019** [The-OpenROAD-Project/OpenROAD-flow-scripts#952]
### Description

I build the docker image in a machine named A and then transfer the image to another machine named B and then run it with `docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd)/flow:/OpenROAD-flow-scripts/flow openroad/flow-$OS_NAME-builder`

get: 
```
groups: cannot find name for group ID 1003
I have no name!@9a41a4f15c50:/OpenROAD-flow-scripts$ echo $USER

I have no name!@9a41a4f15c50:/OpenROAD-flow-scripts$ whoami
whoami: cannot find name for user ID 1003
I have no name!@9a41a4f15c50:/OpenROAD-flow-scripts$ id
uid=1003 gid=1003 groups=1003
```

### Suggested Solution

update the docker start cmd

 from 
```
docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd)/flow:/OpenROAD-flow-scripts/flow openroad/flow-$OS_NAME-builder
```

to 

```
docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd)/flow:/OpenROAD-flow-scripts/flow  -v /etc/passwd:/etc/passwd:ro -v /etc/group:/etc/group:ro openroad/flow-$OS_NAME-builder:latest

```

this is just for ubuntu :)

**Comment by vijayank88** [comment]
@hanm2019 
How you installed OpenROAD-flow-scripts in docker mode?
Did you followed this instruction https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildWithDocker.html ?

**Comment by vijayank88** [comment]
Before running docker command set following in terminal if default steps used for installation:
`export OS_NAME=centos7`

**Comment by hanm2019** [comment]
> 

Yes, I install it in ubuntu

**Comment by hanm2019** [comment]
> 

I run in ubuntu OS, so I set the OS_NAME to `ubuntu20.04` by

```
export OS_NAME=ubuntu20.04
```

**Comment by vijayank88** [comment]
After `export OS_NAME=ubuntu20.04` did you followed below steps for installation?
```
git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
cd OpenROAD-flow-scripts
./etc/DockerHelper.sh create -target=dev -os=$OS_NAME
./etc/DockerHelper.sh create -target=builder -os=$OS_NAME
```
Otherwise docker default built with `centos7` OS

**Comment by hanm2019** [comment]
> After `export OS_NAME=ubuntu20.04` did you followed below steps for installation?
> 
> ```
> git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
> cd OpenROAD-flow-scripts
> ./etc/DockerHelper.sh create -target=dev -os=$OS_NAME
> ./etc/DockerHelper.sh create -target=builder -os=$OS_NAME
> ```
> 
> Otherwise docker default built with `centos7` OS

Yes, and my host OS is ubuntu18.04, but I config the `OS_NAME` to `ubuntu20.04` and after running
```
./etc/DockerHelper.sh create -target=dev -os=$OS_NAME
./etc/DockerHelper.sh create -target=builder -os=$OS_NAME

```
I build the image successfully.

**Comment by rovinski** [comment]
@hanm2019 is this on a machine which has a networked login? e.g. the passwd and group files are stored on the network?

**Comment by hanm2019** [comment]
I build the docker image in a machine named A and then transfer the image to another machine named B. the `groups: cannot find name for group ID 1003` is shown when I run the docker on machine B.

**Comment by hanm2019** [comment]
the image is `by1906007/flow-ubuntu20.04-builder:latest`

**Comment by rovinski** [comment]
Are the `passwd` and `group` files different between machines `A` and `B`? It doesn't really feel like mounting the `passwd` and `group` files is the right solution here - a vast majority of users will never encounter this issue...

**Comment by hanm2019** [comment]
Yeah, the `passwd` and `group` files are different between machine `A` and `B`.

**Comment by vijayank88** [comment]
> I build the docker image in a machine named A and then transfer the image to another machine named B. the `groups: cannot find name for group ID 1003` is shown when I run the docker on machine B.

@hanm2019 above should be part of your issue description. So it will easy to understand the problem and resolved quickly.
Without proper description it leads to long conversation.

**Comment by vijayank88** [comment]
@vvbandeira 
Is mirror copying will work ?
